gradle.properties

```diff
@@ -13,46 +13,26 @@
 # limitations under the License.
 
 group=org.ballerinalang
-version=0.4.1-SNAPSHOT
-ballerinaLangVersion=2201.8.2
+version=0.5.0-SNAPSHOT
+ballerinaLangVersion=2201.8.3
```

Comment:
We need to bump the version to `2201.9.0` and we have to change `distribution = "2201.9.0"` in the package's `Ballerina.toml`, once the update 9 is released